UrlManager.php

```diff
@@ -356,7 +360,7 @@ protected function redirectToLanguage($language)
 
         Yii::$app->getResponse()->redirect($redirectUrl);
         if (YII_ENV_TEST) {
-            throw new \yii\base\Exception($redirectUrl);
+            throw new \yii\base\Exception(\yii\helpers\Url::to($redirectUrl));
```

Comment:
@MetalGuardian All tests still succeed if I remove this. But after your findings I'm a bit unsettled now :D. So maybe you can clarify.